docs/examples/promise/12_upgrade_chain/index.js

```diff
@@ -5,8 +5,7 @@
 const { Api } = require('@cennznet/api');
 
```

Comment:
` : api.tx.consensus.setCode(`0x${code}`); // For previous versions` this code is long removed.

I don't think this example has much value to ordinary users so would prefer to remove it

Reply:
ya.. will remove it..